test/functional/legacy/writefile_spec.lua

```diff
@@ -1,12 +1,14 @@
 -- Tests for writefile()
 
-local helpers = require('test.functional.helpers')(after_each)
+local helpers = require('test.functional.helpers')
```

Comment:
You will need to keep the `(after_each)`. This will error out otherwise (it does so, you can check the quickbild log).